client/setec/client_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package setec_test
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/tailscale/setec/client/setec"
+	"github.com/tailscale/setec/types/api"
+)
+
+const testSecrets = `{
+  "apple": {
+     "secret": {"Version": 1, "Value": "c2F1Y2U="}
+  },
+  "pear": {
+     "unrelated": true
+  },
+  "plum": {
+     "secret": {"Version": 2, "Value": "dGFydGxldA=="}
+  },
+  "cherry": {
```

Comment:
Add a test for `foo/bar/baz` style secrets? We don't treat `/` specially, but it's a convention we're encouraging so would be good to lock it in with a test case if that changes someday.

Reply:
Good call, done.